kubernetes/iac/modules/hw-tf-module-cluster-resources/apisix.tf

```diff
@@ -85,3 +90,48 @@ resource "helm_release" "apisix-dashboard" {
 
   namespace = "apisix"
 }
+
+resource "kubectl_manifest" "apisix_dapr_upstream" {
+  depends_on = [
+    helm_release.apisix
+  ]
+
+  yaml_body = <<YAML
+apiVersion: apisix.apache.org/v2
+kind: ApisixUpstream
+metadata:
+  name: ${local.apisix_name}-dapr
+  namespace: ${local.apisix_namespace}
+spec:
+  loadbalancer:
+    type: roundrobin
+  portLevelSettings:
+  - port: 3500
+    scheme: http
+  - port: 50001
+    scheme: grpc
+YAML
+}
+
+resource "kubectl_manifest" "apisix_dapr_route" {
+  depends_on = [
+    helm_release.apisix
+  ]
+
+  yaml_body = <<YAML
+apiVersion: apisix.apache.org/v2
+kind: ApisixRoute
+metadata:
+  name: ${local.apisix_name}-dapr
+  namespace: ${local.apisix_namespace}
+spec:
+  http:
+  - backends:
+    - serviceName: ${local.apisix_name}-dapr
+      servicePort: 50001
+    match:
```

Comment:
we should restrict this to only the api hosts (`api.hw.de`, `staging-api.hw.de`). this way we can add routes for other hosts (like `kc.hw.de`)